### PR TITLE
etapa 2

### PR DESCRIPTION
**Pull request by carmenacatrinei**

### CODE REVIEW

File: src/ro/catalog/servicii/ServiceCatalog.java [hunk 1]
```diff
@@ -15,16 +18,16 @@ public static Catalog creazaCatalog(int grupa){
         return new Catalog(grupa);
     }
 
-    static public Materie creazaMaterie(String denumire, int nrCredite){
+    static public Materie creareMaterie(String denumire, int nrCredite){
```
Comment (alina-puscasu) [inline src/ro/catalog/servicii/ServiceCatalog.java]: I don't see any audit service being called on these methods from etapa1, you only call it (and write to data.csv) for ServiceUtilizatori, right?

File: data/profesori.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿Dragan,Ioan
```
Comment (alina-puscasu) [inline data/profesori.csv:1]: datele din acest csv corespund entitatii Utilizator, pe care Profesor o extinde, poate ca ar fi fost mai corect sa faci csv-ul pentru utilizatori atunci, sau sa il imbogatesti pentru Profesor sa cuprinda si lista de materii pe care ai declarat-o ca field al entitatii

File: src/ro/catalog/servicii/Audit.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    private Audit() {}
+
+    public static Audit getAudit() {
+        if (auditService == null)
+            auditService = new Audit();
+        return auditService;
+    }
+
+    public void writeAction(String action) throws IOException {
```
Comment (alina-puscasu) [inline src/ro/catalog/servicii/Audit.java]: redundant declaration of throws IOException, since you treat it (declare it in catch)

**Comment by alina-puscasu** [comment]
remove unused imports and unused code in general  (like methods, since you have a lot of methods that are not being used anywhere in your code)

if you want to fix comments for 1p, you can try read also the data from the other 2 csv files and call audit method and write in the audit csv at each method call from all services

Apoi, poti face merge in main si lucra pt etapa 3, fara PR pt etapa 3 pentru ca veti prezenta direct codul vostru sursa de pe localul vostru in demo din 30/31 mai in functie de cum ai ales. Spor, Carmen!

File: src/ro/catalog/servicii/ServiceCatalog.java (hunk 1 above)
Comment (carmenacatrinei) [reply src/ro/catalog/servicii/ServiceCatalog.java]: Yes, that's right, I will solve it.

File: src/ro/catalog/servicii/Audit.java (hunk 3 above)
Comment (carmenacatrinei) [reply src/ro/catalog/servicii/Audit.java]: Fixed.

**Pull request merged**